### PR TITLE
sig-windows - moving 1.29 / 1.30 jobs, adding serial-slow jobs, and adding presubmit jobs

### PR DESCRIPTION
**Pull request by marosset**
/assign @jsturtevant @jeremyrickard 
/sig windows

### CODE REVIEW

File: config/jobs/kubernetes-sigs/sig-windows/release-1.29-windows-presubmits.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    labels:
+      preset-azure-anonymous-pull: "true"
+      preset-azure-capz-sa-cred: "true"
+      preset-azure-cred-only: "true"
+      preset-capz-containerd-1-7-latest: "true"
+      preset-capz-windows-2022: "true"
+      preset-capz-windows-common-pull: "true"
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+      preset-windows-private-registry-cred: "true"
```
Comment (jsturtevant) [inline config/jobs/kubernetes-sigs/sig-windows/release-1.29-windows-presubmits.yaml:32]: is this missing preset `preset-capz-windows-common-129`

Comment (jsturtevant) [reply config/jobs/kubernetes-sigs/sig-windows/release-1.29-windows-presubmits.yaml:32]: scratch that, its a Pre-submit so will be building the latest versions based on the k/k branch

**Review by jsturtevant — APPROVED** [review]
/lgtm
/approve

Thanks for adding the serial slow jobs to the periodics

**Review by jeremyrickard — APPROVED** [review]

**Pull request merged**